lib/js/stanford_publication.behavior.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Behavior Example that works with Webpack.
+ *
+ * @see: https://www.npmjs.com/package/drupal-behaviors-loader
+ *
+ * Webpack wraps everything in enclosures and hides the global variables from
+ * scripts so special handling is needed.
+ */
+
+export default {
+
+  // Attach Drupal Behavior.
+  attach: function (context, settings) {
+    (function ($) {
+
+      // only open and close filter menu when on responsive size.
+      const mediaQuery = window.matchMedia('(max-width: 991px)')
+
+      function handleTabletChange(e) {
+        // Check if the media query is true
+        if (e.matches) {
+
+          // Remove the h2 and replace with button.
+          $("#system-menu-blockstanford-publication-topics-menu")
+            .contents()
+            .unwrap()
+            .wrap('<button class="publication-topics__collapsable-menu"></button>');
+
+          $('.publication-topics__collapsable-menu', context).click(function () {
```

Comment:
One thing you can do to improve this is set the `<button>` element to a variable and then add the click handler to that. ```
const myvar = $('<button/>')
myvar.click(() => { do stuff });
```